vernac/declareUniv.mli

```diff
@@ -16,7 +16,7 @@ exception AlreadyDeclared of (string option * Id.t)
 
 (** Internally used to declare names of universes from monomorphic
    constants/inductives. Noop on polymorphic references. *)
-val declare_univ_binders : GlobRef.t -> UnivNames.universe_binders -> unit
+val declare_univ_binders : GlobRef.t -> Entries.universes_entry *  UnivNames.universe_binders -> unit
```

Comment:
Revert this
It's better to get the declared universes from the global env imo

Reply:
The reason I made it like this is that to possibly later support naming section universes, one needs to be able to access the universes of a section variable. But in the case e.g. of variables, the universes are not attached in the kernel to the variables but declared independently (hence `DeclareUctx.declare_universe_context` in `Declare.declare_variable_core`). So, I see no way to retrieve universes of variables with a global access.

Note that this may work with the changes I'm doing to section variables. If represented as `Const`, they will be equippable with universes.

So, a possibility is to first wait for the work on sections.

Reply:
@SkySkimmer : I finally backtracked on the `DeclareUniv` changes, since this is more related to naming section variables than to this PR per se.